easybuild/easyconfigs/p/Python/Python-2.7.8-goolf-1.5.14.eb

```diff
@@ -0,0 +1,160 @@
+name = 'Python'
+version = '2.7.8'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.8.2'
+scipyversion = '0.14.0'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+dependencies = [
+    ('bzip2', '1.0.6'),          # for bz2 package
+#    ('zlib', '1.2.8'),          # use version selected by libxslt below
+    ('libreadline', '6.2'),
+    ('ncurses', '5.9'),
+    ('LLVM', '3.2'),             # for Numba
+    ('SQLite', '3.8.4.3'),       # for IPython and general use
+    ('libxslt', '1.1.28'),       # for lxml.  brings in libxml2 and zlib dependencies.
+    ('CUDA', '6.0.37','', True), # for pycuda, duh.
+#   ('OpenSSL', '1.0.1f'),       # OS dependency should be preferred for security reasons
+
+    ('libjpeg-turbo','1.3.0'),   # For matplotlib and Pillow
+    ('libpng','1.6.10'),         # For matplotlib and Pillow
+    ('freetype', '2.5.3'),       # For matplotlib and Pillow
```

Comment:
we try to stay away from excluding extra dependencies that are only required for Python packages...

see also the effort in https://github.com/hpcugent/easybuild-easyconfigs/pull/1387 to bundle multiple extensions that required extra dependencies together in a single `-extra` module